wordpress-mobile/WordPress-iOS
# Issues/4032 reader cards

Refs #4032 

This PR adds the new reader card table view cell.  A few notes:
- the new cell is not currently used
- some features (e.g. tags, word count) are stubbed awaiting API and core data changes.
- the feature will not be fully implemented until the new styled list is ready. 

Submitting a PR for code review now in the interest of smaller more contained feature related PRs.  Once the new list and other changes are ready we'll do another PR for its integration. There will also be a few additional style/layout changes later to catch up with recent style changes in the web reader. 

To test in its current state: 
You can apply [this diff](https://gist.github.com/aerych/f58687d0ab695862ea6a) to the bridging header and the current ReaderPostsViewController.  Note that it didn't make sense to wire everything up perfectly since this is a WIP. 

Example card:
![ios simulator screen shot aug 7 2015 12 30 26 pm](https://cloud.githubusercontent.com/assets/1435271/9141692/33ef40c8-3d00-11e5-902d-9c3ed485475c.png)


Needs Review: @jleandroperez
